content/docs/06-integrations/00-antrea-cni.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Antrea CNI'
+metaTitle: 'Antrea CNI'
+metaDescription: 'Antrea CNI network pack for Palette Kubernetes Clusters'
+hiddenFromNav: true
+type: "integration"
+category: ['network']
+logoUrl: 'https://registry.spectrocloud.com/v1/palette-upgrader/blobs/sha256:b6081bca439eeb01a8d43b3cb6895df4c088f80af978856ddc0da568e5c09365?type=image/png'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Antrea CNI
+
+Palette supports Antrea controller network interface (CNI) for Kubernetes clusters. Antrea  CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility. 
+
+Antrea leverages Open vSwitch to implement pod networking and security features. Open vSwitch enables Antrea to implement Kubernetes network policies efficiently.
+
+# Supported Versions
+
+**1.9.x**
+
+## Prerequisites
```

Comment:
```suggestion
# Prerequisites
```

Because we only have one element, these have to get raised by one level. Once we add another major version and add a tab, we decrease all of these by one.